test/integration/conftest.py

```diff
@@ -68,14 +69,22 @@ def is_valid_ipv6(address):
         except ipaddress.AddressValueError:
             return False
 
-    def get_public_ip(ip_version="ipv4"):
+    def get_public_ip(ip_version="ipv4", retries=3):
```

Comment:
```suggestion
    def get_public_ip(ip_version: str = "ipv4", retries: int = 3):
```
Let's add type annotations to it 👍